pkg/thanos/statefulset.go

```diff
@@ -220,6 +220,7 @@ func makeStatefulSetSpec(tr *monitoringv1.ThanosRuler, config Config, ruleConfig
 		},
 		"remote-write-config",
 	)
+	//if version.GTE(minRemoteWriteVersion) && len(tr.Spec.RemoteWrite) > 0 {
```

Comment:
should we remove it?

Reply:
I'll update after I confirmed that the new e2e test fails without it :)